components/insight/SRC/org/openmicroscopy/shoola/agents/metadata/editor/EditorModel.java

```diff
@@ -178,7 +178,6 @@
 	static {
 		EXCLUDED_FILE_NS = new ArrayList<String>();
 		EXCLUDED_FILE_NS.add(FileAnnotationData.COMPANION_FILE_NS);
-		EXCLUDED_FILE_NS.add(FileAnnotationData.FLIM_NS);
```

Comment:
I don't think we will have legacy one.
This work was solely used in Dundee and "turned off" from UI a while ago